packages/cozy-harvest-lib/src/components/cards/LaunchTriggerAlert.jsx

```diff
@@ -65,6 +66,16 @@ export const LaunchTriggerAlert = ({
   const isClisk = konnectorPolicy.name === 'clisk'
   const isKonnectorDisconnected = isDisconnected(konnector, trigger)
 
+  const shouldDisplayRunningAlert = () => {
+    if (isInError) return false
+    if (isInMaintenance) return false
+    if (!isKonnectorRunnable) return false
+    if (isKonnectorDisconnected) return false
+    if (running && konnector.clientSide) return true
+
+    return false
+  }
```

Comment:
Wondering if this should not be inside the policy?

Reply:
I think that could be better yeah. @LucsT what do you think please?

Reply:
@doubleface will have more insights to share I think. I already shared my though here about the policy https://github.com/cozy/cozy-libs/pull/2195#issuecomment-1598259418 

Maybe it can be done next time...

Reply:
Since I already moved the same kind of logic in policies for https://github.com/cozy/cozy-libs/pull/2195, I can do it if this PR is merged before mine